cephadm-purge-cluster.yml

```diff
@@ -11,29 +11,27 @@
 #
 # ansible-playbook -i <inventory host file> cephadm-purge-cluster.yml -e fsid=<your fsid>
 
-- name: confirm whether user really meant to upgrade the cluster
+
+- name: check local prerequisites are in place
   hosts: localhost
-  become: false
+  gather_facts: false
+  become: true
+
   tasks:
     - name: fail if fsid was not provided
       fail:
-        msg: >
+        msg: |
           You must provide the cluster fsid to be purged.
-          ansible-playbook -i <inventory host file> cephadm-purge-cluster.yml -e fsid=<your fsid>
+          e.g. ansible-playbook -i <inventory host file> cephadm-purge-cluster.yml -e fsid=<your fsid>
       when: fsid is undefined
 
     - name: fail if admin group doesn't exist or is empty
       fail:
-        msg: >
+        msg: |
           You must define a group [admin] in your inventory and add a node where
           admin keyring is present at /etc/ceph/admin.client.keyring
       when: "'admin' not in groups or groups['admin'] | length < 1"
 
-- name: make sure admin keyring is present and pause cephadm
```

Comment:
why did you remove this?
the presence of this keyring must be checked on the node admin[0]

Reply:
a sugar low..no idea. thanks for catching. The check is in the wrong place - I had it in the localhost checks!